### PR TITLE
Pop open the terminal and run UCM if we dont find it

### PR DESCRIPTION
**Pull request by tapegram**
Partial implementation of https://github.com/unisonweb/vscode-unison/issues/6

This PR will automatically open the embedded terminal and try to run UCM if we weren't able to find a running process to connect to.

We will only try this 1 time to avoid shenanigans or needing to worry about accidentally getting stuck in weird loops.

This makes no attempt to address the problem if UCM isn't installed at all. Hopefully, this will make it a little clearer that maybe UCM is not on the PATH? It might be nice in the future to show a nice message with a link to installation instructions.

This behavior can be disabled via a configuration (`unison.automaticallyOpenUCM`)

### Happy path + quitting UCM doesn't result in the extension starting it again

https://user-images.githubusercontent.com/1281274/199854295-cf5f86c4-a262-4143-80ce-1044bd2a6ef9.mov

### "Simulating" UCM not being on the PATH

https://user-images.githubusercontent.com/1281274/199854354-d41caf9c-5ddf-4051-97a1-8bf05beb43a9.mov

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -98,5 +104,11 @@
         "typescript": "^4.8.4",
         "vsce": "^2.9.2"
     },
-    "license": "SEE LICENSE IN LICENSE"
+    "license": "SEE LICENSE IN LICENSE",
+    "__metadata": {
```
Comment (tapegram) [inline package.json:108]: no idea where this came from? should I remove it?

File: package-lock.json [hunk 2]
```diff
@@ -1,1646 +1,8 @@
 {
```
Comment (tapegram) [inline package-lock.json:1]: I don't really know what's going on in here. I think I have an older version of npm installed so it set the lockfile version back. Not sure why the packages changed though. LMK if I should revert the changes to this file or something.

**Review by ChrisPenner — APPROVED** [review]
Thanks, this is great!

I made a small tweak to change when messages pop up,
and also added a `ucmCommand` config parameter since I expect the development team will appreciate it.

I'll publish the new version ASAP 👍🏼
